docs/atomic.rst

```diff
@@ -3,12 +3,12 @@
 Atomic operations
 =================
 
-In Taichi, augmented assignments (e.g., ``x[i] += 1``) are automatically `atomic <https://en.wikipedia.org/wiki/Fetch-and-add>`_.
+In Taichi, the augmented assignments (e.g., ``x[i] += 1``) are automatically `atomic <https://en.wikipedia.org/wiki/Fetch-and-add>`_.
```

Comment:
Thanks for proposing this PR! However, I'm not sure if we need a `the` here. I feel like the original form is preferred since we are referring to *all* augmented assignments in Taichi, instead of a *specific group/subset*.

English is not my native language, and I can only feel very subtle differences between the two versions. Maybe we need some help from a native speaker. @KLozes What do you think?  :-)

Reply:
> I also think `the` does not fit quite right here.

Thanks for the inputs!



> But actually, its not even _all_ augmented assignments. Since `*=` and `/=` aren't atomic. We may want to be more explicit about what each augmented assignment corresponds to.

Ahh you are right! I should update this chapter... Thanks for pointing this out!

Reply:
Well, I didn't check the grammar textbook but here is a similar issue I can found real quick. Trust me, we need that 'the' here due to the specified context 'in taichi'. If the context were not specified, the 'the' here becomes redundant but the statement becomes false. 😄

Reply:
Thanks, but I still don't think we need the `the` here. We are trying to refer to all augmented assignments in Taichi instead of a subset of them. If we use `the augmented assignments`, readers may feel like what we are referring to is undefined.

Sorry for being stubborn here, but I would stick to the original expression.

Reply:
Haha, thanks for being stubborn. It's a perfect chance for me to learn. I can confirm from a native English speaker that ''both ways convey the same meaning'' though he preferred the version without 'the'.

Reply:
Thanks for understanding and for confirming with a native speaker. I really like everything else in this PR except for the extra `the`s. Feel free to convert it to `Ready for review` and I'll approve and merge.

Reply:
Sure, there you go. Thanks 😄